actions/databases.go

```diff
@@ -0,0 +1,290 @@
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			log.Println(aerr.Error())
+			return c.Error(400, aerr)
+		} else {
+			log.Println(err.Error())
+			return err
+		}
+	}
+
+	output := struct {
+		*rds.DescribeDBClustersOutput
+		*rds.DescribeDBInstancesOutput
+	}{
+		clustersOutput,
+		instancesOutput,
+	}
+	return c.Render(200, r.JSON(output))
+}
+
+// DatabasesGet gets details about a specific database
+//   If the `all=true` parameter is passed it will return a list of clusters in addition to instances.
+func DatabasesGet(c buffalo.Context) error {
+	// if all param is given, we'll return information about both instances and clusters
+	// otherwise, only database instances will be searched
+	all := false
+	if b, err := strconv.ParseBool(c.Param("all")); err == nil {
+		all = b
+	}
+
+	rdsClient := RDS[c.Param("account")]
+	var clustersOutput *rds.DescribeDBClustersOutput
+	var instancesOutput *rds.DescribeDBInstancesOutput
+	var err error
+	clusterNotFound := true
+
+	if all {
+		// search clusters for the given db name
+		clusterNotFound = false
+		clustersInput := &rds.DescribeDBClustersInput{
+			DBClusterIdentifier: aws.String(c.Param("db")),
+		}
+		clustersOutput, err = rdsClient.Service.DescribeDBClusters(clustersInput)
+		if err != nil {
+			if aerr, ok := err.(awserr.Error); ok {
+				if aerr.Code() == rds.ErrCodeDBClusterNotFoundFault {
+					log.Println(rds.ErrCodeDBClusterNotFoundFault, aerr.Error())
+					clusterNotFound = true
+				} else {
+					log.Println(aerr.Error())
+				}
+			} else {
+				log.Println(err.Error())
+			}
+		}
+	}
+
+	// search instances for the given db name
+	instancesInput := &rds.DescribeDBInstancesInput{
+		DBInstanceIdentifier: aws.String(c.Param("db")),
+	}
+	instancesOutput, err = rdsClient.Service.DescribeDBInstances(instancesInput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			if aerr.Code() == rds.ErrCodeDBInstanceNotFoundFault {
+				log.Println(rds.ErrCodeDBInstanceNotFoundFault, aerr.Error())
+				if clusterNotFound {
+					return c.Error(404, aerr)
+				}
+			} else {
+				log.Println(aerr.Error())
+				return c.Error(400, aerr)
+			}
+		} else {
+			log.Println(err.Error())
+			return err
+		}
+	}
+
+	output := struct {
+		*rds.DescribeDBClustersOutput
+		*rds.DescribeDBInstancesOutput
+	}{
+		clustersOutput,
+		instancesOutput,
+	}
+	return c.Render(200, r.JSON(output))
+}
+
+// DatabasesPost creates a database in a given account
+//   It will create a database instance as specified by the `Instance` hash parameters.
+//   If a `Cluster` hash is also given, it will first create an RDS cluster and the instance next.
+func DatabasesPost(c buffalo.Context) error {
+	type DatabaseCreateInput struct {
+		// https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#CreateDBClusterInput
+		Cluster *rds.CreateDBClusterInput
+		// https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#CreateDBInstanceInput
+		Instance *rds.CreateDBInstanceInput
+	}
```

Comment:
I'd rather see this defined at the top of the file I think unless you make it anonymous